torch/csrc/jit/mobile/function.cpp

```diff
@@ -49,6 +49,15 @@ bool Function::append_operator(
     }
   }
 
+  if (model_version == 0x3L &&
+      model_version < caffe2::serialize::kProducedBytecodeVersion &&
+      opname == c10::OperatorName("aten::_convolution", "")) {
```

Comment:
We probably can consider adding a comment to point to the PR that introduces the schema change, or put the old and new schema in the comment directly.